autowarefoundation/autoware.universe
# chore(object_range_splitter): rework parameters

## Description

Create param file.
Must be merged with https://github.com/autowarefoundation/autoware_launch/pull/821


## Tests performed





- Launched a node individually and confirmed that it loads the parameter as expected
- Launch Autoware with perception_mode = radar, and confirmed that the `split_range` is loaded as 80.0.
- Launch Autoware with perception_mode = lidar_radar_fusion, and confirmed that the `split_range` is loaded as 70.0.
- Launch Autoware with perception_mode = camera_lidar_radar_fusion, and confirmed that the `split_range` is loaded as 70.0.
- 
## Effects on system behavior



No effect expected

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
